docs/usage.rst

```diff
@@ -224,6 +224,29 @@ The one exception to this rule is for trigonometric functions applied to data wi
   >>> np.sin(45.*degree)
   array(0.70710678)
 
+Logarithmic Quantities and Units
+********************************
+
+The logarithmic quantities level-of-power and level-of-field and the units neper and
+bel are supported. In the next example, we represent the power measurements, p, as
+a logarithmic quantity at reference level, p_ref, in the units decibel.
```

Comment:
`p` and `p_ref` need double backticks, right now I think the underscore in p_ref will cause formatting issues

Reply:
done.